oak-run/src/main/java/org/apache/jackrabbit/oak/run/RevisionsCommand.java

```diff
@@ -74,15 +80,18 @@ public class RevisionsCommand implements Command {
 
     private static final Logger LOG = LoggerFactory.getLogger(RevisionsCommand.class);
 
+    private static final int REVISION_CAP = getInteger("oak.revision.cap", 250);
+
     private static final String USAGE = Joiner.on(System.lineSeparator()).join(
             "revisions {<jdbc-uri> | <mongodb-uri>} <sub-command> [options]",
             "where sub-command is one of",
             "  info     give information about the revisions state without performing",
```

Comment:
```suggestion
            "  info        give information about the revisions state without performing",
```
(minor: indentation)

Reply:
Fixed